src/puppetlabs/puppetdb/query.clj

```diff
@@ -233,26 +233,19 @@
 (defn node-columns
   "Return the queryable set of fields and corresponding table names where they reside"
   [version]
-  (case version
```

Comment:
there's a definition in this file for v3-renamed-resource-columns that is now unused, also a related comment